src/containers/Review/ReviewWrapper.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Route, Switch } from 'react-router'
+import { Header } from 'semantic-ui-react'
+import { NoMatch } from '../../components'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { FullStructure } from '../../utils/types'
+
+const ReviewWrapper: React.FC = () => {
+  const {
+    match: { path },
+    query: { reviewId },
+  } = useRouter()
+
+  return (
+    <Switch>
+      <Route exact path={path}>
+        <ReviewHomeNEW />
+      </Route>
+      <Route exact path={`${path}/:reviewId`}>
+        <ReviewPageNEW reviewId={Number(reviewId)} />
+      </Route>
+      <Route exact path={`${path}/:reviewId/summary`}>
+        <ReviewSummaryNEW reviewId={Number(reviewId)} />
```

Comment:
I need to refine the diagram a little bit, I am still thinking that ReviewPageWrapper is needed (which will display ReviewPage or ReviewSummary (but will filter reviewAssignment from the passed list to match reviewId (and if not then 404)

Reply:
I think that is a bit confusing, because it seems like this ReviewPagePageWrapper would be pointing to 2 routes of the router that would otherwise be in ReviewWrapper? I'm confused. After you refine this let me know if we  need to create new issues to do that change. 

I was going to start working on issue #378 now, so should be able to help testing how it works after passing the reviewAssignment to ReviewPage route and on #380 see if can compare assignment with review without problems.

Reply:
I've adjusted application flow diagram just now.

The reason for ReviewPageWrapper:

useGetReviewStructure takes reviewAssignment (not reviewId), because it's used to calculate progress even when review does not exist. So we need to use reviewAssignments from ReviewWrapper (passed as prop to ReviewPageWrapper), and try to find matching reviewAssignment, if found we can render Summary and ReviewPage, and pass through reviewAssignment.